Update data-importer how-to document.

mahendra-mahajan-mendix opened the pull request:
Initial draft of the how-to document for Data Importer. Based on the existing documentation with more details about the individual steps.

quinntracy committed:
Create Data Importer How-to

mahendra-mahajan-mendix committed:
Update data-importer how-to document.

Initial draft of the how-to document for Data Importer. Based on the existing documentation with more details about the individual steps.

MarkvanMents commented on content/en/docs/howto/integration/data-importer:
```diff
@@ -2,3 +2,68 @@ title: "Data Importer Extension"
 url: /howto/integration/data-importer
 weight: 21
 description: "Overview of the Data Importer in Studio Pro"
```
Missing front matter delimiters
```suggestion
description: "Overview of the Data Importer in Studio Pro"
---
```

Also needed as the first line of the doc.

mahendra-mahajan-mendix committed:
Update data-importer - final draft

data importer 'how-to' final draft for release. Pls review and edit as required.

quinntracy committed:
Added front matter

quinntracy committed:
Revised intro and prereqs

quinntracy committed:
Fixed URL

quinntracy committed:
Fixed headers

quinntracy committed:
Revised more sections

quinntracy committed:
Revised for grammar

quinntracy committed:
Reviewed for grammar

quinntracy committed:
Fixed numbering

quinntracy committed:
Grammar fix

quinntracy committed:
Fixed file name

quinntracy committed:
Add screenshots

quinntracy committed:
Updated title and URL

quinntracy committed:
Grammar

quinntracy committed:
More screenshots

quinntracy committed:
More edits

quinntracy committed:
Grammar fixes

quinntracy committed:
Minor edit

quinntracy committed:
Added cross-link reference

quinntracy commented on content/en/docs/howto/integration/data-importer:
```diff
@@ -71,11 +71,7 @@ The Data Importer document creation is complete and can be used to import data i
-{{< figure src="/attachments/refguide/integration/data-importer-extension/data-importer-in-marketplace-module.png" >}}
+Use the Data Importer to periodically import and validate data from an Excel file that is generated by another app or system. 
 
 ### 4.1 Custom Activity {#Import-data-from-file}
```
Can you provide some extra context for the example process below? Is there any additional information that the user may benefit from knowing in regard to the sample?

quinntracy commented on content/en/docs/howto/integration/use-the-data-importer.md:
```diff
@@ -91,15 +91,15 @@ The following are the properties in the **Output** section:
-1. Add a Button on the home page with a caption 'Upload Customer Data'
-2. On click of this button you can create an Object of System.FileDocument and pass the control to another page (UploadCustomerData) where file can be uploaded.
+1. Open the home page and add a button with the caption *Upload Customer Data*.
+2. Click the button and create a `System.FileDocument` object and pass the control to another page (**UploadCustomerData**) where the file can be uploaded.
```
To confirm, for this example, the user would need to add a page to their app called **UploadCustomerData**, correct?

quinntracy commented on content/en/docs/howto/integration/use-the-data-importer.md:
```diff
@@ -91,15 +91,15 @@ The following are the properties in the **Output** section:
-3. On the 'UploadCustomerData' page - include a Data view for FileDocument and include a 'File Manager' to assist a file upload.
-4. Add an Action button to call Microflow which will help us import the input file. The uploaded file will be passed as a parameter to this Microflow.
+3. On the **UploadCustomerData** page, include a data view for the *FileDocument* and include a 'File Manager' to assist with a file upload.
+4. Add an **Action** button to call a microflow, which will import the input file. The uploaded file will be passed as a parameter to this microflow.
```
Should we assume the user knows how to add a parameter or should we add extra information about them?

quinntracy commented on content/en/docs/howto/integration/use-the-data-importer.md:
```diff
@@ -105,25 +105,25 @@ The *Import data from file* custom activity needs an input file to import data f
 
-3. Let's try to address these errors. Double-click the activity and in the **File** field, choose the input file that is passed on from the file upload page to this Microflow as a parameter.
-4. In the **Data importer document** field, click *Select* and choose the Data Importer document you want to use.
+3. To address these errors, double-click the activity and in the **File** field, choose the input file that is passed from the file upload page to this microflow as a parameter.
```
When testing this process, I didn't have the option to choose/select a data importer document from the drop-down. Are there ways to troubleshoot this step?

quinntracy committed:
Implemented feedback

quinntracy committed:
Fixed image links

quinntracy committed:
Add screenshot

quinntracy committed:
Updated header

quinntracy committed:
Add screenshot

quinntracy committed:
Implemented feedback

quinntracy committed:
Updated steps with feedback

quinntracy committed:
Merge branch 'patch-23' of https://github.com/mahendra-mahajan-mendix/docs into pr/7003

quinntracy committed:
Updated with feedback

quinntracy committed:
Grammar fixes

quinntracy committed:
Minor grammar fixes

quinntracy committed:
Revised step numbering
